### DISCUSSION TITLE
Consensus SVs Recommendations?

### DISCUSSION THREAD

**Discussion by WuSelina** [uclahs-cds/pipeline-call-sSV#100] [Q&A]
Hi all,

Since we have both DELLY and Manta SV callers, I'm wondering if it would be helpful to have a standardized approach for finding consensus SVs. 

I ended up using [SURVIVOR](https://github.com/fritzsedlazeck/SURVIVOR) for consensus SV calling for each sample. This returns a VCF for each consensus output using the SURVIVOR VCF format, which keeps some of the DELLY and Manta `INFO` and `FORMAT` fields.

Some parameter adjustments (deviating from defaults):
- Tolerate 10 kbp distance gaps for SVs
- Minumum 100bp size

I am interested in learning how others perform consensus SV calling too, and if you have any feedback. Thank you!

**Comment by maotian06** [comment]
@WuSelina 

I asked similar questions to Helen  @helen-zhu, and here is the summary of her suggestions:

1. There is no standard that would suit all the samples. Based on her previous study experience with Dr. Joachim Weischenfeldt, the lab who built PCAWG consensus SV pipeline, "the trick is literally to plot a ton of examples of structural variants you get between callers and see if the consensus makes sense to you and what the weaknesses are in the disagreements."

2. Here are some resources:
Paper: [https://www.nature.com/articles/s41586-020-1969-6](https://urldefense.com/v3/__https://www.nature.com/articles/s41586-020-1969-6__;!!CjcC7IQ!MUHexzNWKmRTObsklqDP9GoJ-bRnYLbIjurW9sdCVMGjM8JaVCEBJJQEdto0QXbPcLAMEMeGm_Omn7KwsAIrD2Y$) - The description of how SV merging works is in the 100-page supplementary methods file in this paper
Code: [https://bitbucket.org/weischenfeldt/pcawg_sv_merge/src/docker/](https://urldefense.com/v3/__https://bitbucket.org/weischenfeldt/pcawg_sv_merge/src/docker/__;!!CjcC7IQ!MUHexzNWKmRTObsklqDP9GoJ-bRnYLbIjurW9sdCVMGjM8JaVCEBJJQEdto0QXbPcLAMEMeGm_Omn7Kwd8cB6MU$)

3. She also recommends to read the original preprint: [https://www.biorxiv.org/content/10.1101/187609v1.abstract](https://urldefense.com/v3/__https://www.biorxiv.org/content/10.1101/187609v1.abstract__;!!CjcC7IQ!MUHexzNWKmRTObsklqDP9GoJ-bRnYLbIjurW9sdCVMGjM8JaVCEBJJQEdto0QXbPcLAMEMeGm_Omn7KwLvqan84$)

**Comment by raagagrawal** [comment]
@maotian06 @WuSelina this is a great discussion. We should definitely be talking more about consensus SVs and whether they should be used in our analyses. I believe in the PCAWG paper, they used 4 SV pipelines and took consensus as SVs called in at least 2/4.

A point I wanted to bring up is that Manta is a read-only archive now. Does this mean it is discontinued and not getting updates? Does that factor into our decision making on using it in our consensus? 

Additionally, in my conversation with @tyamaguchi-ucla, it came up that Delly's cutoff features perform better with higher coverage samples. So maybe consensus only makes sense with lower coverage samples, where Delly alone might be prefered at a higher cutoff on higher coverage samples.

**Comment by tnyamaguchi** [comment]
>Additionally, in my conversation with @tyamaguchi-ucla, it came up that Delly's cutoff features perform better with higher coverage samples.

Let me clarify. What I meant was that Delly has a hard cutoff for the number of support reads (min_clique_size) to call SVs, and we should adjust it depending on the coverage and purity.

https://github.com/uclahs-cds/pipeline-call-sSV

"""
Minimum number of supporting PE or split-read (SR) alignments required for a clique to be identified as a structural variant by Delly. Adjust this parameter to control the sensitivity and specificity of Delly variant calling. Default set to 5.
"""
